flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/table/UpsertKafkaDynamicTableFactory.java

```diff
@@ -150,6 +152,8 @@ public DynamicTableSource createDynamicTableSource(Context context) {
 
         final BoundedOptions boundedOptions = getBoundedOptions(tableOptions);
 
+        Integer parallelism = tableOptions.get(SCAN_PARALLELISM);
```

Comment:
A trivial comment, I'm not sure if it's appropriate：
what about inlining the line into line 174  ?

Reply:
Putting it as a separate line is probably more readable, WDYT?

Reply:
okay~